jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -378,9 +378,17 @@ public void afterTaskForwardedEvent(TaskEvent event) {
         Task ti = event.getTask();
         TaskPersistenceContext persistenceContext = getPersistenceContext(((TaskContext)event.getTaskContext()).getPersistenceContext());
         try {
-            persistenceContext.persist(new TaskEventImpl(ti.getId(), org.kie.internal.task.api.model.TaskEvent.TaskEventType.FORWARDED, ti.getTaskData().getProcessInstanceId(), ti.getTaskData().getWorkItemId(), userId));
-    
-    
+            StringBuilder message = new StringBuilder();
+            String entitiesAsString = (ti.getPeopleAssignments().getPotentialOwners()).stream().map(oe -> oe.getId()).collect(Collectors.joining(","));
+            message.append("Forward to [" + entitiesAsString + "]");
```

Comment:
Usage of _StringBuilder_  can be replaced by simple _String_  here. It looks to me that StringBuilder is an overkill, but take it as a suggestion.

For example:
String message = String.format("Forward to [%s]", entitiesAsString);

Reply:
The number of entities into getPotentialOwners could be pretty big. Anyway, to be coherence, the rest of messages in other methods of this class are defined like StringBuilder.